packages/calcite-components/src/components/pagination/pagination.tsx

```diff
@@ -123,13 +145,13 @@ export class Pagination
     };
   }
 
-  /**
-   * Made into a prop for testing purposes only
-   *
-   * @internal
-   */
-  // eslint-disable-next-line @stencil-community/strict-mutable -- updated by t9n module
-  @Prop({ mutable: true }) messages: PaginationMessages;
+  @State() maxItems = maxItemBreakpoints.xsmall;
+
+  @State() totalPages: number;
+
+  private resizeObserver = createObserver("resize", (entries) =>
```

Comment:
Shouldn't `breakpoints` be placed before `resizeObserver` or this one moved after `breakpoints`? I'm assuming these are grouped by `private`, `@Element`, and `@State` props.